business/MetadataDatabase/Services/PacsServices.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MetadataDatabase.Controllers;
+using MetadataDatabase.Models;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace MetadataDatabase.Services
+{
+    public class PacsServices: IPacsServices
+    {
+        private static readonly HttpClient client = new HttpClient();
+        private readonly PacsConfiguration settings;
+
+        public PacsServices(IOptions<PacsConfiguration> settings)
+        {
+            this.settings = settings.Value;
+        }
+
+        public async Task<IEnumerable<QidoSeries>> GetSeries()
```

Comment:
namming convension : Async function should be suffixed by Async
rename to GetSeriesAsync()

Reply:
Done